runtimes/huggingface/mlserver_huggingface/codecs/base.py

```diff
@@ -181,6 +186,15 @@ def decode_request(cls, request: InferenceRequest) -> Dict[str, Any]:
 
             value = get_decoded_or_raw(item)
             values[item.name] = value
+        if request.parameters is not None:
+            if hasattr(request.parameters, "extra"):
+                extra = request.parameters.extra
+                if isinstance(extra, dict):
+                    values.update(extra)
+                else:
+                    logging.warn(
+                        "Extra inference kwargs should be kept in a dictionary."
```

Comment:
Could you output the value of the parameter as well in the warning message? And perhaps change the warning message to be in the form of "Extra parameters cannot be parsed, expected a dictionary" to be more descriptive in the message?